pkg/plugin/handlers_query.go

```diff
@@ -85,6 +86,7 @@ func QueryData(ctx context.Context, pCtx backend.PluginContext, backendQuery bac
 			Category:         query.StatsCategory,
 			Fields:           query.StatsFields,
 			GroupBy:          query.StatsGroupBy,
+			Interval:         query.StatsInterval,
```

Comment:
Could you add one test in `handlers_query_test.go` for this case?

Reply:
I added regex validation that the interval format is correct and two tests for correct and incorrect formats.